TACC/Core-CMS-Custom
# Task/WP-746-styles-primary-color-blue

## Overview

Updates @tacc/core-styles package to use expected color

## Related


- [WP-746](https://tacc-main.atlassian.net/browse/WP-746)


## Changes

…

## Testing

1. Stop all containers and rebuild
2. npm ci, npm run build (there will be new warnings on build - this is expected)
3. Run client and make sure no breaking changes in styles across the site
4. Make sure links/primary color is set to blue
5. In console, check that color: var(--global-color-link-on-light--normal); is computed to # 039

## UI

###Select fields as links turn to blue
<img width="1198" alt="Screenshot 2024-11-05 at 2 45 19 PM" src="https://github.com/user-attachments/assets/b6c68abb-aede-4a6e-b202-8fbed0e3e59f">

###Button colors change to blue
<img width="1450" alt="Screenshot 2024-11-05 at 2 45 55 PM" src="https://github.com/user-attachments/assets/179801b9-6a3e-4634-b484-4b7785bc5e08">

###Nav bar is now blue
<img width="439" alt="Screenshot 2024-11-05 at 2 46 25 PM" src="https://github.com/user-attachments/assets/ba97d5b8-c141-40be-b827-6e3a84ed3521">

###Pagination is now blue
<img width="1528" alt="Screenshot 2024-11-05 at 2 47 11 PM" src="https://github.com/user-attachments/assets/0f0c4557-d4cd-4aab-89bd-5c2c418f3627">

## Notes
### All buttons should use core-components/Button rather than Reactstrap component Button in order to pick up styles
<img width="494" alt="Screenshot 2024-11-05 at 2 47 55 PM" src="https://github.com/user-attachments/assets/4d446a79-f926-4ee4-a67f-b4aa6af93fdd">
